content/en/graphing/infrastructure/network_performance_monitor.md

```diff
@@ -49,7 +148,7 @@ The following shows all flows from IP addresses to Security groups. Only flows w
 - `Sent` metrics measure the value of something from the `source` to the `destination`
 - `Received` metrics measure the value of something from the `destination` to the `source` as measured from the source.
 
-Metrics might be different for `some.metric(a to b)` and `some.metric(b to a)` if there is a large number of packet drops on one leg of the trip. In this case, if `b` sends a lot of bytes to `a`, the flows that originate at `b` include those bytes, but the flows that originate at `a` do not see them as received.
+The default collection interval is five minutes and retention is five days. Metrics might be different for `some.metric(a to b)` and `some.metric(b to a)` if there is a large number of packet drops on one leg of the trip. In this case, if `b` sends a lot of bytes to `a`, the flows that originate at `b` include those bytes, but the flows that originate at `a` do not see them as received.
```

Comment:
I'd remove `leg of the trip`. This might not translate well. Maybe replace with `transmittal`.

Reply:
I just removed it all together, because this is true really whenever the packet drops happen.